src/website/app/pages/Theming/theming.md

```diff
@@ -135,10 +158,35 @@ const myTheme = createTheme('dusk', {
 });
 ```
 
+### `withTheme(component, options)`
+
+A higher order component (HOC) that enables theme access inside any component.
```

Comment:
should we link here to react docs, like we do for render props?

https://reactjs.org/docs/higher-order-components.html